en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/http-2.0.md

```diff
@@ -1,16 +1,16 @@
 # HTTP 2.0
```

Comment:
We need to remove this doc. THis is coming from mgw v320 and not applicable for CC v1.

Reply:
fixed in https://github.com/wso2/docs-apim/pull/5032/commits/2b2ceaa1a15bcf07a89001a6684a9824c5eb4a60